tests/test_output.py

```diff
@@ -1814,6 +1850,7 @@ def test_result_list_backup(self, capsys):
 
         backup = find_by_attr(json_output[bi.server_name], "backup_id", bi.backup_id)
         assert bi.backup_id == backup["backup_id"]
+        assert backup["backup_type"] == "rsync"
```

Comment:
```suggestion
```

I think we can remove this as this part is tested in the new unit test (`test_result_list_backup_types`)